src/hash.hpp

```diff
@@ -24,6 +26,40 @@ namespace Simhash {
      * @return hash representative of the content of the text */
     template <typename Hash=jenkins>
     class Simhash {
+    private:
+        class Accumulator {
+        private:
+            int64_t v[64];
+
+            size_t window;
+            Cyclic<hash_t> cyclic;
+
+        public:
+            Accumulator(): window(3), cyclic(window) {
```

Comment:
It's fairly typical to include a space before the colon for the initializer list.